component/http/route.go

```diff
@@ -18,6 +18,26 @@ type Route struct {
 	middlewares []MiddlewareFunc
 }
 
+// GetPath returns route path value.
+func (r Route) GetPath() string {
```

Comment:
The convention in Go for getter is to omit the Get part and fix the comment.
```suggestion
func (r Route) Path() string {
```